test/transition.spec.js

```diff
@@ -620,3 +620,91 @@ test('x-transition supports css animation', async () => {
     )
     expect(document.querySelector('span').classList.contains('animation-leave')).toEqual(false)
 })
+
+test('remaining transitions forced to complete if they exists', async () => {
```

Comment:
I run the same test against master and it did not fail. It seems a bit complex, is it supposed to test this feature?

Reply:
Fixed by improving the test